content/webapp/pages/api/newsletter-signup.ts

```diff
@@ -1,12 +1,13 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
 import { isUndefined } from '@weco/common/utils/type-guards';
-import fetch from 'node-fetch';
 
 const dotdigitalUsername = process.env.dotdigital_username;
 const dotdigitalPassword = process.env.dotdigital_password;
 
 type Status = 'ok' | 'error';
 
-async function createSubscription({
+export async function createSubscription({
```

Comment:
don't think this needs to be exported

Reply:
good point, it won't be used anywhere but here. fixed